src/Components/WebAssembly/Samples/HostedBlazorWebassemblyApp/Server/Startup.cs

```diff
@@ -52,6 +52,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         app.UseStaticFiles();
 
         app.UseRouting();
+        app.Use(async (context, next) =>
+        {
+            context.Response.Headers.Add("Cross-Origin-Embedder-Policy", "require-corp");
+            context.Response.Headers.Add("Cross-Origin-Opener-Policy", "same-origin");
+            await next();
+        });
```

Comment:
Is this change needed for something in this PR? If not it would be good to avoid adding it, since we might accidentally start relying on it for the tests to pass.